Automattic/newspack-blocks
# feat(carousel): rename block and reorganise settings

### All Submissions:






### Changes proposed in this Pull Request:

Similar to #1943, this time for the "Post Carousel" block.

### How to test the changes in this Pull Request:

1. Add a bunch of Carousel Post blocks to a page with various settings
2. Switch to this branch
3. Check if the blocks are still displaying the correct information and has kept the correct settings

### Other information:
